packages/dev/gui/src/2D/controls/displayGrid.ts

```diff
@@ -90,14 +90,14 @@ export class DisplayGrid extends Control {
         this._markAsDirty();
     }
 
-    /** Gets or sets the tickness of minor lines (1 by default) */
+    /** Gets or sets the thickness of minor lines (1 by default) */
     @serialize()
-    public get minorLineTickness(): number {
-        return this._minorLineTickness;
+    public get minorLineThickness(): number {
+        return this._minorLineThickness;
     }
 
-    public set minorLineTickness(value: number) {
-        this._minorLineTickness = value;
+    public set minorLineThickness(value: number) {
+        this._minorLineThickness = value;

```

Comment:
Wheereas I totally agree that this is a typo and should be fixed, this is a breaking change. We will need to add a (deprecated) proxy to support the old  syntax